### ISSUE TITLE
Add support for '3b9f958131fe9f006646530501001171df000000000013' token

### ISSUE THREAD

**Issue by brunohelius** [PeculiarVentures/fortify#158]
Reader name: FS USB Token 0
ATR: 3B9F958131FE9F006646530501001171DF000000000013

```js
{
   "cards": [{
   	"atr": "3B9F958131FE9F006646530501001171DF000000000013",
   	"name": "Feitian ePass2003",
   	"driver": "AA9DFBBE3614005EA8F071425776D9839AFD61AB"
   }],
   "drivers": [{
   	"id": "AA9DFBBE3614005EA8F071425776D9839AFD61AB",
   	"name": "Driver name",
   	"file": {
   		"windows": "path/to/pkcs11.dll",
   		"osx": "path/to/pkcs11.dylib"
   	}
   }]
}
```

[Smart card ATR parsing 3B9F958131FE9F006646530501001171DF000000000013](https://smartcard-atr.appspot.com/parse?ATR=3B9F958131FE9F006646530501001171DF000000000013)

**Comment by rmhrisk** [comment]
@brunohelius can you tell us anything about this token?

- What is the make and model?
- What is the PKCS#11 library you use with it?
- What operating system re you using it with?

It looks like it might be a Feitian ePass2003?

if so this issue (https://github.com/PeculiarVentures/fortify/issues/146) and yours would likely be resolved once we know what PKCS#11 library works with it.

If it really is an ePass2003 it might work with the OpenSC middleware according to this (https://github.com/OpenSC/OpenSC/wiki/Feitian-ePass2003).


If so you can update cards.json with:

```
{
   "cards": [{
   	"atr": "3B9F958131FE9F006646530501001171DF000000000013",
   	"name": "Feitian ePass2003",
   	"driver": "993988460d8f49a2ac519a2935f11533"
   }]
}
```

And it might work.

**Comment by microshine** [comment]
Duplicate issue https://github.com/PeculiarVentures/fortify/issues/146
